### ISSUE TITLE
Opening email showing pure HTML code - 7.9.1

### ISSUE THREAD

**Issue by PhillyD1973** [salesagility/SuiteCRM#3783]
#### Issue
When opening an email.  The body contains HTML code rather than the email.  Some of the original emails seems to be in the 2nd body tag.

#### Expected Behavior
email should open in a readable format.

#### Actual Behavior
Email is opening with a ton of HTML code in the Body

![image](https://user-images.githubusercontent.com/29067877/27503890-2e4abfd2-584f-11e7-98a2-44a1679dc6f3.png)

#### Steps to Reproduce

1.  Open inbound email

#### Your Environment
* SuiteCRM Version used: 7.9.1
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Chrome (MAC) Version 58.0.3029.110 (64-bit)
* Environment name and version (e.g. MySQL, PHP 7): MySQL PHP 5.4
* Operating System and version (e.g Ubuntu 16.04): Unknown (hostgator)

**Comment by blloyd78** [comment]
I've had this problem too - if you refresh the page, the email should be displayed correctly.

**Comment by Mausino** [comment]
@PhillyD1973 @blloyd78 the issue also persist on my site ...  refresh helps :) (solves the issue)

1. if you open email in detaiview
2. if you import email

**Comment by tentandtable** [comment]
Hello,

I have found the solution for this issue,It worked for me.
Replace display function in view.detail.php in Emails module.
Path : modules/Emails/views/view.detail.php
```
public function display()
    {
        parent::display();
        switch ($this->bean->status) {
            case 'unread':
                $this->bean->status = 'read';
                $this->bean->save();
                break;
            default:
                break;
        }
    }
```

**Comment by PreetiPeshwani** [comment]
Hello,

I have found the solution for this issue,It worked for me.
Replace display function in view.detail.php in Emails module.
Path : modules/Emails/views/view.detail.php
```
public function display()
    {
        parent::display();
        switch ($this->bean->status) {
            case 'unread':
                $this->bean->status = 'read';
                $this->bean->save();
                break;
            default:
                break;
        }
    }
```

**Comment by Dillon-Brown** [comment]
We haven’t been able to replicate this issue in any newer version of SuiteCRM such as 7.10.x or 7.11.x. This issue has been closed as version 7.8.x of SuiteCRM has reached its end of life and therefore will no longer be receiving updates. If you feel this issue is still present in the supported versions please leave a comment explaining the steps to reproduce and we'll re-open this issue.

For more information about 7.8.x end of life please visit our [blog post](https://suitecrm.com/7-8-x-end-of-life/)
